users/tests.py

```diff
@@ -1,11 +1,18 @@
+from http import client
+import json, jwt
+
 from unittest.mock import patch, MagicMock
+
+from django.conf   import settings
+from django.conf   import settings
 from django.test   import TestCase, Client
 
-from users.models  import User
+from users.models  import *
+from places.models import *
 
 class KakaoLoginTest(TestCase):
```

Comment:
이 클래스는 아래와 무슨 차이가 있는건가요?
```suggestion
class KakaoLoginTest(TestCase):
```